src/validations/date-marks/best-date.js

```diff
@@ -5,10 +5,9 @@
     valid = "The date mark must 20 characters or less.";
     return;
   }
-  if (/[|&;$%"<>()\{\}+\\]/.test(input)) {
-    valid = 'Please do not include html or special characters e.g. not any of these |&;$%"<>(){}+\/';
+  if (!/^\d{1,2}\/\d{1,2}\/\d{4}$/.test(input)) {
```

Comment:
There is request to remove this, we don't need this validation. Please refer to this ticket's- C4A-521 comments

Reply:
Remove the validation? This is to stop injection, I believe that this should allow a date format but not other sql / mongosql injections.

Reply:
Not removing per say but to modify, as the date field can have string of characters which is not in format 11/11/21 but can also be 'Apr 2020'

Reply:
:+1:  I didn't understand, maybe not enough :coffee: